src/oumi/core/configs/training_config.py

```diff
@@ -203,3 +203,12 @@ def __post_init__(self):
                     dataset_params.dataset_kwargs["processor_kwargs"] = {
                         **self.model.processor_kwargs
                     }
+
+        # Verl will error without a validation dataset.
+        if (
+            self.training.trainer_type == TrainerType.VERL_GRPO
+            and not self.data.validation.datasets
+        ):
+            raise ValueError(
+                "At least one validation dataset is required for VERL GRPO training."
```

Comment:
```suggestion
                "At least one validation dataset is required for VERL_GRPO training."
```
Minor nit for consistency with other messages